.github/workflows/codesee-arch-diagram.yml

```diff
@@ -6,6 +6,7 @@ on:
     types: [opened, synchronize, reopened]
     branches-ignore:
       - 'renovate/**'
+      - 'i18n-sync-**'
```

Comment:
@nhcarrigan  This did not work for the renovate PRs, though 😕

Reply:
You are correct. Will PR a new approach.